Formula/harbour.rb

```diff
@@ -2,7 +2,7 @@ class Harbour < Formula
   desc "Portable, xBase-compatible programming language and environment"
   homepage "https://harbour.github.io"
   license "GPL-2.0"
-  revision 1
+  revision 2
```

Comment:
I rebuilt `harbour` a couple days ago (#107019) so can probably drop commit, rebase, and retry

Reply:
👍